### ISSUE TITLE
prosper_version forgets version on reinstall

### ISSUE THREAD

**Issue by lockefox** [EVEprosper/ProsperCommon#15]
Though prosper_version tools work fine for clean installs, reinstall causes issues.  Git checking is causing problems in dependent Travis jobs where it looks up the "current running version" rather than `version.txt`

```bash
Running ProsperCommon-1.1.1/setup.py -q bdist_egg --dist-dir /tmp/easy_install-fokg7r3h/ProsperCommon-1.1.1/egg-dist-tmp-wx98dp8f
package init file 'prosper/__init__.py' not found (or not a regular file)
zip_safe flag not set; analyzing archive contents...
prosper.common.__pycache__._version.cpython-35: module references __file__
prosper.common.__pycache__.prosper_config.cpython-35: module references __file__
prosper.common.__pycache__.prosper_logging.cpython-35: module references __file__
creating /home/travis/build/EVEprosper/ProsperBots/.eggs/ProsperCommon-0.1.0-py3.5.egg
Extracting ProsperCommon-0.1.0-py3.5.egg to /home/travis/build/EVEprosper/ProsperBots/.eggs
Installed /home/travis/build/EVEprosper/ProsperBots/.eggs/ProsperCommon-0.1.0-py3.5.egg
Traceback (most recent call last):
  File "setup.py", line 165, in <module>
    'fast': QuickTest
  File "/opt/python/3.5.3/lib/python3.5/distutils/core.py", line 148, in setup
    dist.run_commands()
  File "/opt/python/3.5.3/lib/python3.5/distutils/dist.py", line 955, in run_commands
    self.run_command(cmd)
  File "/opt/python/3.5.3/lib/python3.5/distutils/dist.py", line 974, in run_command
    cmd_obj.run()
  File "/home/travis/virtualenv/python3.5.3/lib/python3.5/site-packages/setuptools/command/test.py", line 203, in run
    installed_dists = self.install_dists(self.distribution)
  File "/home/travis/virtualenv/python3.5.3/lib/python3.5/site-packages/setuptools/command/test.py", line 194, in install_dists
    ir_d = dist.fetch_build_eggs(dist.install_requires)
  File "/home/travis/virtualenv/python3.5.3/lib/python3.5/site-packages/setuptools/dist.py", line 446, in fetch_build_eggs
    replace_conflicting=True,
  File "/home/travis/virtualenv/python3.5.3/lib/python3.5/site-packages/pkg_resources/__init__.py", line 858, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'ProsperCommon~=1.1.1' distribution was not found and is required by the application
```

**Comment by lockefox** [comment]
maybe add something like 

```python
if 'site-packages' in HERE:
    version = file_version
```

**Comment by lockefox** [comment]
Fixed in v1.1.2
